projects/storefrontstyles/scss/cxbase/blocks/forms.scss

```diff
@@ -220,7 +220,7 @@ input[type='radio'] {
   display: inline-block;
   border-style: solid;
   border-width: 2px;
-  border-color: var(--cx-color-light);
+  border-color: var(--cx-color-medium);
```

Comment:
Changes to existing styles require corresponding migration doc.
https://github.com/SAP/spartacus/blob/develop/docs/migration/6_0/styling.md